### PR TITLE
Update default translations for email and sms

### PR DESCRIPTION
**Pull request by IniZio**
ref #3935

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -112,6 +112,11 @@ push-image:
 
 .PHONY: html-email
 html-email:
+	go run ./scripts/generatemjml/main.go \
+		-i resources/authgear/mjml/templates \
+		-t resources/authgear/mjml/translations \
+		-o resources/authgear
+
```
Comment (louischan-oursky) [inline Makefile:119]: - Store .mjml.gotemplate next to .mjml.
- Store translation.json next to .mjml.gotemplate

Comment (IniZio) [reply Makefile:119]: You mean prefer keeping the generated `.mjml` in repo and placing `.mjml.gotemplate` next to them?

Comment (louischan-oursky) [reply Makefile:119]: Yes.

File: resources/authgear/templates/translations/en.json [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    "Code": "{{ .Code }}",
+    "Disclaimer": "Don't share it with anyone. If you didn't trigger this setup, please ignore this email."
+  },
+  "SetupSecondaryOOBSMSTXT": {
+    "Title": "Setup two-step verification on {{ template \"app.name\" }} with this one-time password {{ .Code }}.",
+    "Disclaimer": "Don't share it with anyone."
+  },
+  "VerificationEmailTXT": {
+    "Title": "Email Verification",
+    "Body": "This email is sent to verify {{ .Email }} on {{ template \"app.name\" }}. Use this code in the verification page.\n{{ .Code }}",
```
Comment (louischan-oursky) [inline resources/authgear/templates/translations/en.json:155]: We have an extra `\n{{ .Code }}` at the end. Remove it.

File: resources/authgear/templates/translations/zh-HK.json [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+  },
+  "ForgotPasswordEmailTXT": {
+    "Title": "重設密碼",
+    "Body": "你要求了重設 {{ template \"app.name\" }} 上的密碼。請按此連結重設密碼。",
+    "Link": "{{ .Link }}",
+    "Disclaimer": "如你未要求重設，則無需理會此電郵。"
+  },
+  "ForgotPasswordSMSTXT": {
+    "Title": "按此連結重設你在 {{ template \"app.name\" }} 的密碼。如你未要求重設，則無需理會此訊息。",
+    "Disclaimer": "Ignore this if you are not resetting.",
```
Comment (louischan-oursky) [inline resources/authgear/templates/translations/zh-HK.json:108]: This should be in Chinese.

File: resources/authgear/templates/translations/zh-TW.json [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+  },
+  "ForgotPasswordEmailTXT": {
+    "Title": "重設密碼",
+    "Body": "你要求了重設 {{ template \"app.name\" }} 上的密碼。請按此連結重設密碼。",
+    "Link": "{{ .Link }}",
+    "Disclaimer": "如你未要求重設，則無需理會此電郵。"
+  },
+  "ForgotPasswordSMSTXT": {
+    "Title": "按此連結重設你在 {{ template \"app.name\" }} 的密碼。如你未要求重設，則無需理會此訊息。",
+    "Disclaimer": "Ignore this if you are not resetting.",
```
Comment (louischan-oursky) [inline resources/authgear/templates/translations/zh-TW.json:108]: This should be in Chinese.

File: Makefile (hunk 1 above)
Comment (louischan-oursky) [reply Makefile:119]: I mean the following structure

```
/authgear
  /templates
    /en
      /messages
         forgot_password_sms.txt
         translation.json
```

**Pull request merged**